docs/docs/60-new-docs/40-operator-guide/20-advanced-installation/20-advanced-with-argocd.md

```diff
@@ -2,8 +2,149 @@
 sidebar_label: With Argo CD
 ---
 
-# Advanced Installation with Argo CD
+# Installation with Argo CD
 
-:::warning
-Placeholder
+This section outlines a few generalized approaches to installing and managing Kargo with non-default configuration options using Argo CD.
+
+:::note
+This section assumes that you have already installed any dependencies or prerequisites required for running Kargo on a Kubernetes cluster. Please refer to [Basic Installation](../../operator-guide/basic-installation#prerequisites) for more details.
+:::
+
+All methods described here will involve deploying Kargo using an Argo CD
+`Application` resource that is configured to obtain Kargo's Helm chart directly
+from its official repo. We will demonstrate a variety of ways to specify
+your own configuration values using `api.adminAccount.passwordHash` and `api.adminAccount.tokenSigningKey` as examples since you are _required_ to
+provide values for these anyway (unless
+[the admin account is disabled instead](../40-security/10-secure-configuration.md#disabling-the-admin-account)).
```

Comment:
```suggestion
--------------------------------------------------------------------------------
All methods described here will involve deploying Kargo using an Argo CD
`Application` resource that is configured to obtain Kargo's Helm chart directly
from its official repo. We will demonstrate a variety of ways to specify your
own configuration values using `api.adminAccount.passwordHash` and
`api.adminAccount.tokenSigningKey` as examples since you are _required_ to
provide values for these anyway (unless
[the admin account is disabled instead](../40-security/10-secure-configuration.md#disabling-the-admin-account)).
```